modules/ROOT/pages/sync/adding-sync-to-your-app.adoc

```diff
@@ -1,6 +1,6 @@
 include::{partialsdir}/attributes.adoc[]
 
-= Using the {sync-service} service SDK
+= Using the {sync-service} SDK
```

Comment:
not the way we refer to SDK in doc, but will fix in post

Reply:
Ah.. Really sorry. Missing tons of context here.